Makefile

```diff
@@ -0,0 +1,23 @@
+.PHONY: build release install uninstall clean test doc reindent
```

Comment:
There is no `doc` target but if listing it here does no harm we can leave it, since this is just a customized version of our standard jbuilder `Makefile`.